.github/workflows/l2geth_ci.yml

```diff
@@ -27,7 +27,7 @@ jobs:
       run: |
         make nccc_geth
   build-geth: # build geth with circuit capacity checker
-    if: github.event_name == 'push' # will only be triggered when pushing to main & staging & develop & alpha
+    if: github.event.pull_request.draft == false # will only be triggered when pushing to main & staging & develop & alpha
```

Comment:
will you revert it back?
otherwise building it every time takes a lot of time

Reply:
yes.